includes/admin/import/class-batch-import-downloads.php

```diff
@@ -106,6 +106,21 @@ public function process_step() {
 					}
 				}
 
+				// Format the date properly
+				if ( ! empty( $args['post_date'] ) ) {
+
+					$timestamp = strtotime( $args['post_date'] );
```

Comment:
No, the 2nd parameter is if you only want the timestamp to be relative to the 2nd time. Supplying it would not give us the proper date.